README.md

```diff
@@ -196,12 +196,15 @@ PUT
 - Checks if user email address exists in database. Adds test score to applicant entry in database.
 - To add an applicant's test score to the database, the request should be sent as a PUT request to /api/aptiudeScore in
```

Comment:
i think there's a typo in the URL right?

Reply:
Resolved